### PR TITLE
[fix](spill) fix wrong disk usage of spill

### PR DESCRIPTION
**Pull request by jacktengg**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by jacktengg** [comment]
run buildall

File: be/src/vec/spill/spill_stream.h [hunk 1]
```diff
@@ -51,7 +51,7 @@ class SpillStream {
 
     const std::string& get_spill_dir() const { return spill_dir_; }
 
-    size_t get_written_bytes() const { return writer_->get_written_bytes(); }
+    auto get_written_bytes() const { return total_written_bytes_; }
```
Comment (yiguolei) [inline be/src/vec/spill/spill_stream.h:54]: 返回值不要用auto

File: be/src/vec/spill/spill_writer.h [hunk 2]
```diff
@@ -44,7 +44,7 @@ class SpillWriter {
 
     int64_t get_id() const { return stream_id_; }
 
-    size_t get_written_bytes() const { return total_written_bytes_; }
+    auto get_written_bytes() const { return total_written_bytes_; }
```
Comment (yiguolei) [inline be/src/vec/spill/spill_writer.h:47]: not use auto in return value type

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run performance

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run performance

**Review by yiguolei — APPROVED** [review]

**Review by mrhhsg — APPROVED** [review]
lgtm

**Pull request merged**